tools/targetbrowser/client.go

```diff
@@ -0,0 +1,59 @@
+package targetbrowser
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+type Client struct {
+	apiKey     string
+	baseURL    string
+	HTTPClient *http.Client
+}
+
+// NewClient creates new trilio backend client with given API key
```

Comment:
it's just generates new HTTP client right.. update description

Reply:
done